pkgs/servers/unfs3/default.nix

```diff
@@ -29,5 +29,8 @@ stdenv.mkDerivation rec {
     license = lib.licenses.bsd3;
     platforms = lib.platforms.unix;
     maintainers = [ ];
+
+    # https://github.com/unfs3/unfs3/issues/13
+    broken = true;
```

Comment:
might be worthwhile to try to apply https://github.com/unfs3/unfs3/pull/17 as a patch, and  see if it works.

Unfortunate that upstream doesn't seem to be very active

Reply:
I saw that but I thought it was too much code (for me) to review. If someone else doesn't show up and review/test this I think is better to let upstream handle it and mark unfs3 as broken.